tests/util/translator/translator.spec.tsx

```diff
@@ -222,323 +222,38 @@ describe("Translator", () => {
     });
 
     describe("parseSchema", () => {
```

Comment:
add one more test to cover the check in conditional rendering which skips applying a rule if the source field is not present

Reply:
added